acyclic/test/src/acyclic/BaseCycleTests.scala

```diff
@@ -53,12 +52,12 @@ object CycleTests extends TestSuite {
         test("innercycle") - make("success/pkg/innercycle")
       }
     }
-    test("self") - make("../../src", extraIncludes = Nil)
+    test("self") - make(s"../../$srcDirName", extraIncludes = Nil)
     test("force") - {
       test("warn") - {
         test("fail") - {
           make("force/simple", force = true, warn = true).exists {
```

Comment:
Depends what `make` does. If `make` performs the assert internally, then no need to wrap in an assert. If `make` returns a `Boolean`, then you need to wrap in an assert

Reply:
`make` returns a `Seq[(String, String)]` (and throws an exception if compilation fails) and then the call to `.exists` returns a `Boolean`, so it looks like we do need `assert`. Updated to add it and to also make sure that we're matching against the uppercase string `"WARNING"` since [the severity levels use uppercase strings](https://github.com/scala/scala/blob/2.13.x/src/reflect/scala/reflect/internal/Reporting.scala#L167-L169)